src/main/java/jimmy/mcgymmy/logic/parser/ParserUtil.java

```diff
@@ -54,9 +54,6 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
     public static Name parseName(String name) throws ParseException {
         requireNonNull(name);
         String trimmedName = name.trim();
-        if (!Name.isValidName(trimmedName)) {
-            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
-        }
```

Comment:
Why was this removed?

Reply:
that code and this one have the same functionality:
![image](https://user-images.githubusercontent.com/58925632/98247909-31318980-1faf-11eb-967b-97aaf0608bc5.png)
So I just remove 1 of them

Reply:
Might be better to leave it in for defensive programming?